src/kili/adapters/kili_api_gateway/asset/formatters.py

```diff
@@ -0,0 +1,24 @@
+"""Formatters for assets retrieved from Kili API."""
+
+import json
+from typing import Dict
+
+from kili.domain.types import ListOrTuple
+
+ASSET_JSON_FIELDS = ("jsonMetadata", "labels.jsonResponse", "latestLabel.jsonResponse")
+
+
+def load_asset_json_fields(asset: Dict, fields: ListOrTuple[str]) -> Dict:
+    """Load json fields of an asset."""
+    if "jsonMetadata" in fields:
+        asset["jsonMetadata"] = json.loads(asset.get("jsonMetadata", "{}"))
+    if "labels.jsonResponse" in fields:
+        asset_labels = asset.get("labels", [])
+        for label in asset_labels:
+            if (label.get("jsonResponse")) is not None:
+                label["jsonResponse"] = json.loads(label["jsonResponse"])
+    if "latestLabel.jsonResponse" in fields and asset.get("latestLabel") is not None:
+        latest_label = asset["latestLabel"]
+        if (latest_label.get("jsonResponse")) is not None:
+            latest_label["jsonResponse"] = json.loads(latest_label["json_response"])
```

Comment:
jsonresponse is a mandatory field, but yes we need to check for latestLabel 

```suggestion
        asset["latestLabel"]["jsonResponse"] = json.loads(asset["latestLabel"]["json_response"])
```